src/BinSkim.Rules/PERules/BA2022.SignSecurely.cs

```diff
@@ -103,7 +103,7 @@ public override void Analyze(BinaryAnalyzerContext context)
             for (uint i = 0; i < signatureCount; i++)
             {
                 string hashAlgorithm, hashEncryptionAlgorithm;
-                winTrustData = this.InitializeWinTrustDataStruct(filePath, WinTrustDataKind.EnforcePolicy, i);
+                winTrustData = this.InitializeWinTrustDataStruct(filePath, WinTrustDataKind.Normal, i);
```

Comment:
>Normal [](http://example.com/codeflow?start=92&length=6)

this is the change that can make the user file pass, but I am not sure of the impact yet,